layouts/includes/sidebar.jade

```diff
@@ -13,6 +13,7 @@ aside.sidebar
       ul
         li: a(href="/develop/hello-world-explained" class=document.url === 'develop/hello-world-explained'? "current" : "") Hello World Explained
         li: a(href="/develop/1-embed-webview/ios" class=document.url === 'develop/1-embed-webview/ios'? "current" : "") Embedding the Webview
+        li: a(href="/develop/push-notifications" class=document.url === 'develop/push-notifications'? "current" : "") Push Notifications
```

Comment:
Fortunately the new nav will not need all of these updates to separate files each time a new menu item needs to be added! 

This all looks great overall. I went through each step myself too and it worked perfectly. I was extra nitpicky on wording and pointing things out for clarity since this topic is popular and can be confusing. My only other suggestion would be to add a screenshot or two for android to this guide. 

Also, the [PhoneGap CLI Reference Guide](http://docs.phonegap.com/references/phonegap-cli) and [PhoneGap Developer App Reference Guide](http://docs.phonegap.com/references/developer-app) will need to be updated for this at least a little bit to mention the new CLI command/property and to ensure people know this feature is now available with the minimum required versions.